build/Publish.targets

```diff
@@ -180,9 +180,9 @@
     <Error Text="Missing expected packages from PackageToPublish. These were defined as expected PackageArtifact's in artifacts.props: %0A - @(_MissingPackageArtifact, '%0A - ')" Condition="@(_MissingPackageArtifact->Count()) != 0" />
   </Target>
 
-  <Target Name="CopyToPublishArtifacts" DependsOnTargets="_CheckFilesToPublish">
-    <MakeDir Directories="$(ArtifactsDir)files\" />
-    <Copy SourceFiles="@(FilesToPublish)" DestinationFolder="$(ArtifactsDir)files\" />
+  <Target Name="CopyToPublishArtifacts" DependsOnTargets="">
+    <MakeDir Directories="$(ArtifactsDir)$([System.IO.Path]::GetDirectoryName($([System.String]::new('%(FilesToPublish.RelativeBlobPath)'))))" Condition="'%(FilesToPublish.RelativeBlobPath)' != ''" />
```

Comment:
Wouldn't `Copy` take care of creating the dir structure?

Reply:
Yup, i always forget it does that. Updated